app/views/layouts/application.html.erb

```diff
@@ -47,6 +47,13 @@
         <% end %>
       <% end %>
 
+      <% if !test_bbb_connection %>
```

Comment:
You are sending too many requests to BBB. You can better test the server availability once and assign its state to a global variable @bbb_available = [true | false]